.changeset/violet-laws-hang.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": patch
+---
+
+Improve inline discount test precision and reliability in draft orders
```

Comment:
Please write changeset where you clearly explain what you have improved. Something like

_Now, test that checks discount calculation asserts against fixed precision (2 decimals)_